fedora-defconfig

```diff
@@ -0,0 +1,6 @@
+# These are required to boot and use a Fedora system installed with
+# default settings
+CONFIG_DM_THIN_PROVISIONING=y
+CONFIG_XFS_FS=y
+
+CONFIG_IOSCHED_BFQ=y
```

Comment:
Why not add it to `base-defconfig`?

Reply:
agree with @ujfalusi . I use the same setup for Ubuntu and Fedora, if we have different modes I'll screw up :-)

BTW I already use SOF on 3 Fedora 34 devices and never had any issue - is this with Fedora 35?

Reply:
I'll move to base-config.

This was Fedora-Server-netinst-x86_64-34-1.2, a small image that I used to save time downloading and flashing a bigger USB stick. I assumed only the default package selection made a difference but there's apparently more to it. In any case these options seem fairly basic and could be used by any distro / flavour IMHO